<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConvertDialog</class>
 <widget class="QDialog" name="ConvertDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Convert</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>12</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>12</number>
   </property>
   <property name="bottomMargin">
    <number>12</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Convert</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,1,0,0,0">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Output format</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboFormat">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>epub</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>kepub</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>mobi</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>azw3</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Output folder</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="textOutputDir"/>
       </item>
       <item>
        <widget class="QCheckBox" name="checkOverwrite">
         <property name="text">
          <string>Overwrite existing files</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkStk">
         <property name="text">
          <string>Send to Kindle</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkDebug">
         <property name="toolTip">
          <string>Prepare archive with details for debugging</string>
         </property>
         <property name="text">
          <string>Debug</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>fb2c</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Path to fb2c executable</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="textConverterPath"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>fb2c config file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="textConverterConfig"/>
       </item>
       <item>
        <widget class="QLabel" name="labelEditConfig">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;https://null&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;Edit config file&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="labelDownloadConverter">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;https://github.com/rupor-github/fb2converter/releases/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;Download fb2converter&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConvertDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConvertDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>labelEditConfig</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>ConvertDialog</receiver>
   <slot>onEditConfig(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>labelDownloadConverter</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>ConvertDialog</receiver>
   <slot>onDownloadConverter(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onEditConfig(QString)</slot>
  <slot>onDownloadConverter(QString)</slot>
  <slot>onToolOutputDir()</slot>
  <slot>onToolConverterPath()</slot>
  <slot>onToolConverterConfig()</slot>
 </slots>
</ui>
